src/containers/Characters.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Cards from '../components/DeckOfCards';
+import { getItems } from '../services/hey-arnold-api';
+
+export default class Characters extends Component {
+  constructor(props) {
+    super(props);
+  
+    this.state = {
+      data: []
+    };
+  }
```

Comment:
since youre not passing anything into this class. i dont think you NEED to have a constructor here. you could just initiate state.